packages/face-detection/src/definitions.ts

```diff
@@ -34,23 +78,438 @@ export interface ProcessImageResult {
+   * Positive euler X is the face is looking up.
+   *
+   * @since 5.1.0
+   */
+  headEulerAngleX: number;
+  /**
+   * Returns the rotation of the face about the vertical axis of the image.
+   * Positive euler y is when the face turns toward the right side of the image that is being processed.
+   *
+   * @since 5.1.0
+   */
+  headEulerAngleY: number;
+  /**
+   * Returns the rotation of the face about the axis pointing out of the image.
+   * Positive euler z is a counter-clockwise rotation within the image plane.
+   *
+   * @since 5.1.0
+   */
+  headEulerAngleZ: number;
+
+  /**
+   * Returns a value between 0.0 and 1.0 giving a probability that the face is smiling.
+   *
+   * @since 5.1.0
+   */
+  smilingProbability?: number;
+  /**
+   * Returns a value between 0.0 and 1.0 giving a probability that the face's left eye is open.
+   *
+   * @since 5.1.0
+   */
+  leftEyeOpenProbability?: number;
+  /**
+   * Returns a value between 0.0 and 1.0 giving a probability that the face's right eye is open.
+   *
+   * @since 5.1.0
+   */
+  rightEyeOpenProbability?: number;
+}
+
+/**
+ * Rect holds four integer coordinates for a rectangle.
+ *
+ * @since 5.1.0
+ * @see https://developer.android.com/reference/android/graphics/Rect.html
+ */
+export interface Rect {
+  /**
+   * The X coordinate of the left side of the rectangle
+   *
+   * @since 5.1.0
+   */
+  left: number;
+  /**
+   * The Y coordinate of the top of the rectangle
+   *
+   * @since 5.1.0
+   */
+  top: number;
+  /**
+   * The X coordinate of the right side of the rectangle
+   *
+   * @since 5.1.0
+   */
+  right: number;
+  /**
+   * The Y coordinate of the bottom of the rectangle
+   *
+   * @since 5.1.0
+   */
+  bottom: number;
+}
+
+/**
+ * Represent a face landmark.
+ * A landmark is a point on a detected face, such as an eye, nose, or mouth.
+ *
+ * @since 5.1.0
+ * @see https://developers.google.com/android/reference/com/google/mlkit/vision/face/FaceLandmark
+ */
+export interface FaceLandmark {
+  /**
+   * Gets the FaceLandmark.LandmarkType type.
+   *
+   * @since 5.1.0
+   */
+  type: LandmarkType;
+  /**
+   * Gets a 2D point for landmark position, where (0, 0) is the upper-left corner of the image.
+   *
+   * @since 5.1.0
+   */
+  position: PointF;
+}
+
+/**
+ * Represent a face contour.
+ * A contour is a list of points on a detected face, such as the mouth.
+ *
+ * @since 5.1.0
+ * @see https://developers.google.com/android/reference/com/google/mlkit/vision/face/FaceContour
+ */
+export interface FaceContour {
+  /**
+   * Gets the FaceContour.ContourType type.
+   *
+   * @since 5.1.0
+   */
+  type: ContourType;
+  /**
+   * Gets a list of 2D points for this face contour, where (0, 0) is the upper-left corner of the image.
+   *
+   * @since 5.1.0
+   */
+  points: PointF[];
+}
+
+/**
+ * PointF holds two float coordinates
+ *
+ * @since 5.1.0
+ * @see https://developer.android.com/reference/android/graphics/PointF.html
+ */
+export interface PointF {
```

Comment:
```suggestion
export interface Point {
```

Let's rename this to `Point`. `PointF` might confuse many and there is no other interface named `Point` anyway.